### PR TITLE
refactor: remove unused helpers and rename utils

### PR DESCRIPTION
**Pull request by matthieuauger**
Remove duplicated code in tests by factorizing context initialization

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

**Review by ClementWalter — APPROVED** [review]

**Review by ClementWalter — APPROVED** [review]

**Comment by matthieuauger** [comment]
@ClementWalter ressource tests seem to fail here, how does it work? Is it normal that test fails when I only changes tests in the PR? thanks

File: tests/cairo_files/instructions/test_push_operations.cairo [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+// Local dependencies
+from utils.utils import Helpers
+from kakarot.model import model
+from kakarot.stack import Stack
+from kakarot.execution_context import ExecutionContext
+from kakarot.instructions.push_operations import PushOperations
+from tests.utils.utils import TestHelpers
+
+@external
+func test__exec_push1_should_add_1_byte_to_stack{
```
[the remark this answers is not in the document]
Comment (matthieuauger) [reply tests/cairo_files/instructions/test_push_operations.cairo:19]: Will update it in next PR

File: tests/cairo_files/instructions/test_push_operations.cairo [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
+}() {
+    alloc_locals;
```
[the remark this answers is not in the document]
Comment (matthieuauger) [reply tests/cairo_files/instructions/test_push_operations.cairo:22]: I think I had a weird issue with one of the _ptr if I didn't add the alloc_locals, will try removing it for the next push (16-32)

File: tests/cairo_files/instructions/test_push_operations.cairo [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    let ctx: model.ExecutionContext* = TestHelpers.init_context_with_bytecode(15, 0xFF);
+    let result = PushOperations.exec_push14(ctx);
+    TestHelpers.assert_stack_last_element_contains(result.stack, 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFF);
+
+    return ();
+}
+
+@external
+func test__exec_push15_should_add_15_byte_to_stack{
```
[the remark this answers is not in the document]
Comment (matthieuauger) [reply tests/cairo_files/instructions/test_push_operations.cairo:188]: Yes! I add an issue that I need to debug starting from push16, I'm merging the current push opcodes for now

**Pull request merged**